packages/hackathon/manifest.lkml

```diff
@@ -0,0 +1,58 @@
+application: hackathon {
+  label: "Hackathon"
+  url: "https://localhost:8080/dist/bundle.js"
+  # file: "bundle.js"
+  entitlements: {
+    local_storage: no
+    navigation: yes
+    new_window: yes
+    new_window_external_urls: [
+      "https://*.looker.com/*",
+      "https://*.google.com/*",
+      "https://*.bit.ly/*",
+      "https://*.imgur.com/*",
+      "https://*.slack.com/*",
+      "https://*.github.com/*",
+      "https://*.youtube.com/*",
+      "https://*.youtu.be/*",
+      "https://*.vimeo.com/*"
+    ]
+    use_form_submit: yes
+    use_embeds: no
+    use_iframes: no
+    use_clipboard: no
+    external_api_urls: ["http://localhost:8081/*"]
```

Comment:
I think this is only needed for the auth token server, which we don't need any more